libshaderc/src/shaderc_private.h

```diff
@@ -33,6 +33,8 @@ struct shaderc_spv_module {
   size_t num_errors = 0;
   // Number of warnings.
   size_t num_warnings = 0;
+  // Compilation result.
+  shaderc_compilation_result compilation_result;
```

Comment:
compilation_status
Also, explicitly set it to something.  Suggest shaderc_compilation_status_null_result_module.  :-)